src/components/JobsPage/ListAllJob.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { Component } from "react";
+import { IoMdArrowRoundDown } from "react-icons/io";
+import { IoMdArrowForward } from "react-icons/io";
+import { IoLogoUsd } from "react-icons/io";
+import { injectIntl, FormattedMessage } from "react-intl";
+import Pagination from "react-js-pagination";
+import { connect } from "react-redux";
+
+class ListALLJob extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activePage: 1,
+      pageRangeDisplayed: 5,
+      data: [],
+      pagination: {}
+    };
+
+    this.handlePageChange = this.handlePageChange.bind(this);
+  }
+
+  componentDidMount() {
+    this.props
+      .http({
+        url: "auth/post/get",
+        method: "GET"
+      })
+      .then(res => {
+        this.setState({
+          data: res.data.response.posts,
+          pagination: res.data.response.pagination
+        });
+      });
+  }
+
+  handlePageChange(pageNumber) {
+    this.props
```

Comment:
this.props
      .http({
        url: "auth/post/get,
        params: {
            page: pageNumber
        }
        method: "GET"
      })
      .then(res => {
        this.setState({ data: res.data.response.posts });
      });

Reply:
Sử dụng cách viết trên